cogs/invite.py

```diff
@@ -180,6 +180,11 @@ def _invite_requests_enabled(self, guild):
     @commands.bot_has_permissions(create_instant_invite=True)
     @basedbot.has_permissions("invite.request")
     async def invite_request(self, ctx, *, reason=None):
+        """
+        Request an invite to the server
+
+        Users that are not verified have to request an invite-link. An admin will then approve this request.
```

Comment:
Verified/not verified is server specific. Whether someone can create or request invites is only dependent on the defined permission.

Since this command is hidden anyways if someone can't request an invite due to permissions, we might as well drop that entire line.

Reply:
Done!